### DISCUSSION TITLE
有人有 rife vapoursynth 的備份嗎？

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [hooke007/mpv_PlayKit#374] [Q&A 问答] [answered]
https://github.com/HomeOfVapourSynthEvolution/VapourSynth-RIFE-ncnn-Vulkan
看起來似乎不知為何，被作者刪除了 QAQ

**Comment by hooke007** [comment, marked as answer]
如果你是指源码，那没有。
有替代品fork https://github.com/styler00dollar/VapourSynth-RIFE-ncnn-Vulkan

**Reply by JustSimplyKyle** [reply]
啊謝謝，這也行~

**Reply by hooke007** [reply]
相关文档已更新
